.github/workflows/publish.yml

```diff
@@ -0,0 +1,139 @@
+name: Publish
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  contracts:
+    runs-on: token-studio-linux-medium
+    container:
+      image: node:20.17.0-alpine3.20 # Using alpine for a smaller image
+    permissions:
+      contents: read
+
+    steps:
+      # * Initial steps
+      - name: Checkout repository
```

Comment:
The Harden Runner step is missing from a lot of files. Is there any reason for that?

Reply:
@mishomihov00 the HArden Runner step is in the "initial-steps/action.yaml" which is imported in all the other files. Isn't it enough?

Reply:
I think it's enough, but I want @rbarkerSL to confirm.

NOTE: The initial-steps action is **NOT** imported in the **version.yaml** workflow!

Reply:
It is the second step after checking out the repository. It is the first step of the action "Initial_Steps" in the .github/actions folder. Do you need this step to be executed before all the steps? The "Initial_Steps" action can be executed before the checkout step.

version.yaml it is fixed now